1_fetch/src/1_MendotaPull.R

```diff
@@ -0,0 +1,20 @@
+## This script pulls down model evaluation data for Lake Mendota
```

Comment:
No need to prefix source scripts with numbers, in my opinion, and we don't generally do it in DS. This is because the precise ordering of functions is less important once we've layered on the phase-level organization and especially once we add in the declarative structure for targets (which is directed graph, for which **targets** chooses the order in which to run things such that it's compatible with the graph).